src/apollo/schema.graphql

```diff
@@ -1,17 +1,18 @@
+"""Generic scalar type respresenting a JSON object"""
 scalar JSONObject # this is a custom type used to handle JSON data
 schema {
   query: Query
   subscription: Subscription
 }
 
 type Query {
-  runsList: [Run]!
-  runMetadata(runIDs: [ID]!): [Run!]!
-  runTrackingData(runIDs: [ID]!, showDiff: Boolean = false): [TrackingDataset]! # showDiff as true will return all tracked fields amongst compared runs
+  runsList: [Run!]!
+  runMetadata(runIds: [ID!]!): [Run!]!
+  runTrackingData(runIds: [ID!]!, showDiff: Boolean! = false): [TrackingDataset!]! # showDiff as true will return all tracked fields amongst compared runs
```

Comment:
I'm not sure `showDiff` needs to be required here since we're setting a default value for it.

In your PR description I see you wrote, "Also had to add some '!' to highlight the need of the fields as they are not optional on BE." Maybe they should be changed on the BE then?